src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -55,5 +55,8 @@
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2" />
     <InternalsVisibleTo Include="NuGet.PackageManagement.Test" />
     <InternalsVisibleTo Include="NuGet.SolutionRestoreManager.Test" />
+    <InternalsVisibleTo Include="NuGet.CommandLine.FuncTest" />
+    <InternalsVisibleTo Include="Msbuild.Integration.Test" />
+    <InternalsVisibleTo Include="NuGet.Commands.FuncTest" />
```

Comment:
Needing `InternalsVisibleTo` for a test proejct other than the test project of the same name as the csproj being modified, is a significant sign of possible tech debt.

Having a quick look over the tests from these three test projects, it's not clear to me why any of those test projects need access to NuGet.PackageManagement internals, especially since this PR/feature didn't add any public APIs to the NuGet.PackageManagement assembly.